omnigibson/utils/motion_planning_utils.py

```diff
@@ -34,16 +44,18 @@ def state_valid_fn(q):
 
     # set lower and upper bounds
     bounds = ob.RealVectorBounds(2)
-    bounds.setLow(-7.0)
-    bounds.setHigh(7.0)
+    map = og.sim.scene._seg_map
+    map_bound = (map.map_size / 2) * map.map_resolution
+    bounds.setLow(-map_bound)
+    bounds.setHigh(map_bound)
```

Comment:
Let's not do this - the map is a static and potentially buggy source of data. We want this to work in arbitrary scenes. Replace this with the AABB union of all the "floors" objects in the scene